From acaa48c3e75f93d25abfd8e2cb78d25558d0cc8f Mon Sep 17 00:00:00 2001
From: Guilhem Saurel <guilhem.saurel@laas.fr>
Date: Thu, 20 Feb 2025 11:38:16 +0100
Subject: [PATCH] disable unittests

---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e767907c8d..f661159f4a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -484,7 +484,6 @@ add_subdirectory(python)
 # ----------------------------------------------------
 # --- UNIT TEST --------------------------------------
 # ----------------------------------------------------
-add_subdirectory(unittest)
 
 pkg_config_append_libs(${PROJECT_NAME})
 pkg_config_append_cflags("-I${PYTHON_INCLUDE_DIRS}")
-- 
2.53.0

